web/src/app/pages/edit-survey/edit-details/edit-details.component.scss

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2024 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.survey-details-container {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  .survey-detail-card {
+    background: #FCFDF7;
+    border: 1px #E2E3DD solid;
+    border-radius: 8px;
+    padding: 24px;
+
+    .card-title {
+      font-size: 16px;
+      font-weight: 700;
+    }
+
+    .card-row {
+      font-size: 14px;
+      font-weight: 600;
+      margin-top: 12px;
+
+      :nth-child(1) {
+        flex: 1;
+      }
+
+      &.delete-survey {
+        display: flex;
+        align-items: center;
+
+        button {
+          border-radius: 100px;
+          border: 1px solid #747775;
+          color: #BA1A1A;
```

Comment:
Is the there a variable we can refer to here and above for color codes?

Reply:
that specific red is used only there